js/curation-tool.js

```diff
@@ -36,10 +36,10 @@ var vm = new Vue({
         },
 
         // UI elements.
-        selected_phylogeny: null,
-        selected_phyloref: null,
-        selected_specifier: null,
-        selected_tunit: null,
+        selected_phylogeny: undefined,
+        selected_phyloref: undefined,
+        selected_specifier: undefined,
+        selected_tunit: undefined,
 
```

Comment:
We do need this here because Vue installs getters and setters on these properties so it can detect when their values change. This allows us to directly assign to this property later with `vm.selected_phyloref = ...`, which will automatically update the user interface to reflect the newly selected phyloreference. They're also handy as a list of all our user interface controlling variables!